crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -46,29 +44,17 @@ def prepare
     end
   end
 
-  def services
```

Comment:
I'm confused why this (and its corresponding tests) was removed in this commit?  Is it really related to the switch to class methods?